### PR TITLE
Craig 538: access group dynamic policy form refactor

### PR DESCRIPTION
**Pull request by aisaly**

### CODE REVIEW

**Review by jvallexm — DISMISSED** [review]

**Review by hjweds — DISMISSED** [review]

File: src/components/forms/access-groups/AccessGroupDynamicPolicyForm.js [hunk 1]
```diff
@@ -30,42 +25,29 @@ class AccessGroupDynamicPolicyForm extends React.Component {
...
   render() {
-    let conditionOperatorGroups = [];
-    eachKey(conditionOperators, (key) => {
-      conditionOperatorGroups.push(conditionOperators[key]);
-    });
     return (
       <>
         <IcseFormGroup>
           <IcseNameInput
             id="name"
```
Comment (jvallexm) [inline src/components/forms/access-groups/AccessGroupDynamicPolicyForm.js:47]: `field` not needed here, remove

File: src/components/forms/access-groups/AccessGroupDynamicPolicyForm.js [hunk 2]
```diff
@@ -30,42 +25,29 @@ class AccessGroupDynamicPolicyForm extends React.Component {
...
             componentName="dynamic_policies"
             field="name"
-            labelText="Name"
+            forceKebabCase={true}
```
Comment (jvallexm) [inline src/components/forms/access-groups/AccessGroupDynamicPolicyForm.js:50]: `={true}` not needed, change to `forceKebabCase`

File: src/components/forms/access-groups/AccessGroupDynamicPolicyForm.js [hunk 3]
```diff
@@ -79,7 +61,7 @@ class AccessGroupDynamicPolicyForm extends React.Component {
               content:
                 "How many hours authenticated users can work before refresh",
             }}
-            formName="expiration"
+            formName="dynamic_policies"
```
Comment (jvallexm) [inline src/components/forms/access-groups/AccessGroupDynamicPolicyForm.js:64]: form name should have `this.props.data.name` as par

**Comment by jvallexm** [comment]
needs unit tests

File: src/components/forms/access-groups/AccessGroupDynamicPolicyForm.js (hunk 3 above)
Comment (aisaly) [reply src/components/forms/access-groups/AccessGroupDynamicPolicyForm.js:63]: can you explain this? just change to
`formName="this.props.data.name"` ?

Comment (jvallexm) [reply src/components/forms/access-groups/AccessGroupDynamicPolicyForm.js:63]: `formName={this.props.data.name + "-dynamic-policies"}`

**Pull request merged**